### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by saranshbarua**

### CODE REVIEW

**Comment by saranshbarua** [comment]
@darkHarry @vaibhavk  PTAL.

**Comment by vnzongzna** [comment]
Looking good,

**Review by vnzongzna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+## Contributing
+
+- Fork this repo and then clone the repo with `git clone https://github.com/osdc/harbor.git` or you can use ssh too.
+
+- Now open chrome, naviagate to **Options** > **More tools** > **Extensions**
+
+- Click on load unpacked and select harbor. Voila your extension is ready!
+
+- In case of any dilemmas you can create an issue or reach us at our [slack channel](https://jiit-lug.slack.com/).
```
Comment (vnzongzna) [inline README.md]: the given link will ask people to sign in, you can use http://bit.ly/jiit-lug link to directly add them in the channel

**Comment by saranshbarua** [comment]
@vaibhavk check out now.

**Pull request merged**